docker/gateway/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-FROM eclipse-temurin:11-jre-focal
+FROM openjdk:17-alpine
```

Comment:
Why openjdk:17-alpine whereas we switch to eclipse-temurin few months ago ?
(Same question for all the other changes here after)